bokeh/util/serialization.py

```diff
@@ -112,11 +112,16 @@ def traverse_data(datum, is_numpy=is_numpy, use_numpy=True):
         use_numpy: toggle numpy as a dependency for testing purposes
     """
     is_numpy = is_numpy and use_numpy
-    if is_numpy and not any(isinstance(el, (list, tuple)) for el in datum):
+    if is_numpy and not any(isinstance(el, (list, tuple, set)) for el in datum):
+        # NOTE: this is necessary, because later .tolist() is used and it will
+        # keep the original, non-serializable set type. Yes, .tolist() doesn't
```

Comment:
I'm confused. `.tolist` is a method on `ndarray`, not `set`.  Just for my own edification can you provide a minimal test case that demonstrates this claim?